tests/wpt/web-platform-tests/performance-timeline/po-observe.any.js

```diff
@@ -33,9 +37,8 @@
   }, "Filter unsupported entryType entryType names within the entryTypes sequence");
 
   async_test(function (t) {
-  var observer = new PerformanceObserver(
+    var observer = new PerformanceObserver(
       t.step_func(function (entryList, obs) {
-        assert_equals(observer, obs, "observer is second parameter");
```

Comment:
Let's make this the only change to these tests for now, please. The other changes are unrelated to #18285.

Reply:
OK